src/feed_the_troll/servers.py

```diff
@@ -167,7 +167,9 @@ def _start_server(self, server_name, server_configuration):
         reconfigure_module = importlib.import_module(server_configuration['module'])
         reconfigure_server_namespace = namespace_from_configuration(server_name, server_configuration)
         if 'overrides' in server_configuration:
-            rospy.set_param(reconfigure_server_namespace, server_configuration['overrides'])
+            parameters = rospy.get_param(reconfigure_server_namespace)
```

Comment:
This is actually not thread save, it could be that new parameters get loaded onto the paramserver in between get_param and set_param